slkspec/core.py

```diff
@@ -50,18 +67,19 @@ class SLKFile(io.IOBase):
         retrieve data from tape.
     override: bool, default: False
         Override existing files
-    touch: bool, default: True
+    touch: bool, default: False
         Update existing files on the temporary storage to prevent them
-        from being deleted.
+        from being deleted. // not necessary as they are read.
```

Comment:
Okay, I would then clarify this a bit more.
```suggestion
Update `mtime` of temporary files to prevent them from being deleted. Depending on the implemented method to delete temporary files this might not be necessary or have no effect.
```

Reply:
implemented